### ISSUE TITLE
Template Studio wizard for non UWP apps?

### ISSUE THREAD

**Issue by degant** [Microsoft/WindowsTemplateStudio#319]
As per the documentation, the wizard and the templates aim to accelerate Universal Windows Platform (UWP) development. 

> Windows Template Studio is a Visual Studio 2017 Extension that accelerates the creation of new Universal Windows Platform (UWP) apps using a wizard-based experience. 

- Is there a plan to also include templates to help kick-start `Xamarin` and `Xamarin.Forms` development? Thinking along the same lines, wouldn't it be great to have the ability to create a Xamarin Forms solution, pick which MVVM framework to use, choose between a hamburger menu or a tabs based app, include settings, themes (light-dark), add maps, login page, splash screen etc. Essentially everything that the Template Studio offers for UWP apps.
- Similarly, it could also be extended to WPF or maybe even MVC Web App development?
- Pardon me for thinking out loud on this one. How about considering such advanced templates for Web API's? It could have options to pick and choose integration with various services. I'm aware that since VS2015  there are already options available to configure Application Insights, authentication (Windows, Azure AD etc) while creating a new API. But wouldn't it be great to also include what people generally spend time doing every time they set up a new Web API, like maybe swagger (or swashbuckle) for documentation instead of the built-in Help pages, integration with some sort of cache (choose between Redis and others), integration with key vault for storing/retrieving secrets. These are just a few scenarios that I could think off the top of my head, but I'm sure every developer and organization would have different services/packages to integrate to every time they set up a new Web API project.

TL;DR Is there anything planned for non-UWP apps or is this project intended to target _only_ Universal Windows Platform apps?

**Comment by ScarlettCode** [comment]
I'd like to second this for Xamarin forms, especially with xaml standard coming, it seems that Xamarin and UWP are going to converge over time so would be nice to be ready for that

**Comment by mrlacey** [comment]
Windows Template Studio has been created to address a request of developers building UWP apps and so it focuses just on that audience. We recognize that there is interest in extending this to other platforms and so wish to capture and record that interest. 
As a new product we don't want to spread ourselves to thinly or try and do too much too soon. We can't yet say how things are likely to progress but by adding specifics about what you want and why can only help.

As this is a general request I'm going to close it and add a [specific issue for Xamarin.Forms](#321). There is already another [issue to track interest in WPF](#316). If there are other platforms you'd like support for please open new issues for each one to help us track interest in different areas.
Please also add to those issues with details of what you'd like to see. There will be differences between what's available for UWP and so to know which specific frameworks, pages, feature and other functionality you're specifically interested in will help with planning and prioritization.

**Comment by degant** [comment]
My intention was mostly to determine if WTS was built only to target UWP or open to more platforms, and that answers my question. It definitely makes sense to not try to target too many too soon. Thanks!

**Comment by crutkas** [comment]
how is this different than #229?
